README.md

```diff
@@ -1,4 +1,17 @@
-# ifk-articles
+<h1 align="center">ingenjorsarbete-for-klimatet.github.io</h1>
 
-News articles and popularized presentations of ongoing and finalized work, for
-review by developing parties before submission.
+<p align="center">
+    <a href="https://github.com/Ingenjorsarbete-For-Klimatet/ingenjorsarbete-for-klimatet.github.io/blob/main/.github/workflows/build.yaml">
+        <img src="https://github.com/Ingenjorsarbete-For-Klimatet/ingenjorsarbete-for-klimatet.github.io/blob/main/.github/workflows/build.yaml/badge.svg?branch=main" alt="build" style="max-width: 100%;">
```

Comment:
The URL isn't working. That is true for any of the `img src` URLs. What should the badge represent?

Reply:
I will have a look again after dinner, but my thought was to include dynamic badges to the template - obviously, it didn't work as I had planned/hoped. :smiley